MOBILE_API.xml

```diff
@@ -509,60 +521,37 @@
     <enum name="MediaClockFormat">
         <element name="CLOCK1">
             <description>
-                minutesFieldWidth = 2;minutesFieldMax = 19;secondsFieldWidth = 2;secondsFieldMax = 99;maxHours = 19;maxMinutes = 59;maxSeconds = 59;
-                used for Type II and CID headunits
+                Deprecated
```

Comment:
Even if something is deprecated we shouldn't remove the description. Since everything is backwards compatible it is still important to know what each thing is.